test/fixtures/flow/type-annotations/135/actual.js

```diff
@@ -0,0 +1,3 @@
+type A = {
+	...any,
```

Comment:
Yeah, no worries. Per your initial comment, the typing support is incomplete, but eventually we will also support spreading `string` (implicitly promotes to `String` instance, which has no own properties).